test/ret/schema_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule Ret.SchemaTest do
```

Comment:
Confused what this test is testing. Seems like it tests DummySchema and DummyStruct, but those only appear to be used in this test. Was this just to test that stream_data was working? Wonder if we should drop the test now that it is.

Reply:
Ah nevermind. It's for testing our schema.ex utilities. Any advantage to using DummySchema/DummyStruct instead of a real one?

Reply:
Good question. `DummySchema` is in fact a real Ecto schema, and `DummyStruct` is a real struct. By using these dedicated data structures instead of other in-use schemas and structs we decouple this module from other unrelated modules. It would be troubling if a rename to `Avatar` for instance were to require a change to this test. As a fringe benefit we can easily move `Schema` into another project or out into its own library.